recipes/LibriSpeech/ASR/transformer/README.md

```diff
@@ -20,7 +20,9 @@ installed in your environment (see extra-requirements.txt)**
 |:-------------:|:---------------------------:| :-----:| :-----:| :-----:| :--------:|
 | 24-03-22 | transformer.yaml | 2.27 | [HuggingFace](https://huggingface.co/speechbrain/asr-transformer-transformerlm-librispeech) | [GoogleDrive](https://drive.google.com/drive/folders/1Nv1OLbHLqVeShyZ8LY9gjhYGE1DBFzFf?usp=sharing) | 4xV100 32GB |
 | 24-03-22 | conformer_small.yaml | 2.49 (**only 13.3M parameters**) | [HuggingFace](https://huggingface.co/speechbrain/asr-conformersmall-transformerlm-librispeech) | [GoogleDrive](https://drive.google.com/drive/folders/1I4qntoodHCcj1JNbDrfwFHYcLyu1S5-l?usp=sharing) | 1xV100 32GB |
-| 06-12-23 | train_hf_whisper.yaml | 3.60 | Not Avail. | Not Avail. | 1xA100 40GB |
+| 06-12-22 | train_hf_whisper.yaml | 3.60 | Not Avail. | Not Avail. | 1xA100 40GB |
+| 27-03-23 | train_hyperconformer_8M.yaml | 2.55  (**only 7.9M parameters**) | NA | [GoogleDrive](https://drive.google.com/drive/folders/1XR0DLQJHLG5U3MWP1HDvqUvc4VqM60dq?usp=drive_link) | 1xP40 24GB |
```

Comment:
I will upload your google drive link to DropBox. I will give you the new link asap to change this one. Thanks again for releasing the weights.

Reply:
https://www.dropbox.com/sh/8jc96avmivr8fke/AABrFEhtWy_3-Q7BHhkh0enwa?dl=0

Reply:
Please make sure that everything is in the DropBox (to double check)